macros/models/_sector/transfers/transfers_native.sql

```diff
@@ -0,0 +1,182 @@
+{% macro transfers_native(blockchain, traces, transactions, native_token_address, genesis_balances=null, native_erc_transfers=null) %}
+
+WITH
+
+native_transfers  as (
+        SELECT 
+            'receive' as transfer_type, 
```

Comment:
Why the receive / send model versus "one row per transfer"?

I worry that this will make it tough to avoid double counting versus the existing transfers models (and would break all existing eth transfers models).

Reply:
responded in pr conversation.